README.md

```diff
@@ -98,6 +98,22 @@ This project uses [CMake] and [Hunter] package manager.
    sudo make install
    ```
 
+#### OpenCL support on Linux
```

Comment:
It's in section `Building from source` where Hunter is enabled by default as far as I can see.

> If you disable hunter, there are different requirements I assume

I think if you use OpenCL you still need this even if Hunter is off.